{"name": "test04", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 2\n  +1\n  -2", "trace": "test01 = 1\n\n    @pytest.mark.ui('nihao')\n    def test04(test01):\n>       assert  test01==2\nE       assert 1 == 2\nE         +1\nE         -2\n\ntest_04.py:16: AssertionError"}, "start": 1625643748095, "stop": 1625643748095, "uuid": "5f4e96fb-e0da-41b8-8f44-7ffda8ace3f2", "historyId": "63cd1041340e63e155aae2627d65ed04", "testCaseId": "7ca89cdf5e59584a1d0636baaec116aa", "fullName": "test_04#test04", "labels": [{"name": "tag", "value": "ui('nihao')"}, {"name": "suite", "value": "test_04"}, {"name": "host", "value": "DESKTOP-I9LDLDD"}, {"name": "thread", "value": "24928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_04"}]}